vllm/model_executor/models/config.py

```diff
@@ -285,22 +297,23 @@ def verify_and_update_config(cls, vllm_config: "VllmConfig") -> None:
         attn_page_size = \
             cache_config.block_size * attn_page_size_1_token
 
-        assert attn_page_size >= mamba_page_size
+        assert attn_page_size >= static_cache_page_size
 
-        if attn_page_size == mamba_page_size:
+        if attn_page_size == static_cache_page_size:
             # don't need to pad mamba page size
             return
 
         # pad mamba page size to exactly match attention
         if (cache_config.mamba_page_size_padded is None
                 or cache_config.mamba_page_size_padded != attn_page_size):
             cache_config.mamba_page_size_padded = (attn_page_size)
-            mamba_padding_pct = 100 * (attn_page_size -
-                                       mamba_page_size) / mamba_page_size
+            static_cache_padding_pct = 100 * (
+                attn_page_size -
+                static_cache_page_size) / static_cache_page_size
             logger.info(
-                "Padding mamba page size by %.2f%% to ensure "
-                "that mamba page size and attention page size are "
-                "exactly equal.", mamba_padding_pct)
+                "Padding fixed state page size by %.2f%% to ensure "
+                "that fixed state page size and attention page size are "
+                "exactly equal.", static_cache_padding_pct)
```

Comment:
"fixed state" vs. "static cache". I guess we should be consistent with terminology here.